### PR TITLE
add Azure.ResourceManager.ServiceLinker.Tests

### LINKED ISSUE #28685: Initial Azure.ResourceManager.ServiceLinker
Service team asked for beta release

### PR DESCRIPTION
**Pull request by ChenTanyi**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by fengzhou-msft** [comment]
Add a `ci.yaml` file and you can refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/ci.yml

**Comment by ChenTanyi** [comment]
> Add a `ci.yaml` file and you can refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/ci.yml

@fengzhou-msft I think the `ci.yaml` has been already added. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicelinker/ci.yml

File: sdk/servicelinker/Azure.ResourceManager.ServiceLinker/tests/Tests/WebAppKeyVaultConnectionTests.cs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        }
+
+        [SetUp]
+        public async Task Init()
+        {
+            await InitializeUserTokenClients();
+        }
+
+        [TestCase]
+        public async Task WebAppKeyVaultConnectionCRUD()
```
Comment (ArthurMa1978) [inline sdk/servicelinker/Azure.ResourceManager.ServiceLinker/tests/Tests/WebAppKeyVaultConnectionTests.cs:29]: In general, we should test List

Comment (ChenTanyi) [reply sdk/servicelinker/Azure.ResourceManager.ServiceLinker/tests/Tests/WebAppKeyVaultConnectionTests.cs:30]: Added `List service linker` in CRUD test

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**